db/class_test.go

```diff
@@ -179,8 +179,8 @@ func TestCreateClass(t *testing.T) {
 	obj.D = ptr
 	require.NoError(t, err)
 
-	CreateTestUser(t, &obj, 0)
-	CreateTestClass(t, &obj, 0, 0)
+	//CreateTestUser(t, &obj, 0)
```

Comment:
Were these failing? Or is commenting out the delete calls enough

Reply:
Yeah, no, they're all fine.